_cucumber/features/support/hooks.rb

```diff
@@ -59,6 +59,11 @@
 end
 
 def user_logout
-  visit("https://it-developers.stage.redhat.com/auth/realms/rhd/protocol/openid-connect/logout?redirect_uri=#{@redirect_url}%3Fredirect_fragment%3D!")
+  if Capybara.app_host.include?('stage') || Capybara.app_host.include?('docker')
```

Comment:
I'm suggesting that the code be changed to 

```
if Capybara.app_host.equals?('developers.redhat.com')
  visit("https://developers.redhat.com/auth/realms/rhd/protocol/openid-connect/logout....
else
  visit("https://it-developers.stage.redhat.com/auth/realms/rhd/protocol/openid-connect/logout....
```

logic is "for live, logout here. For everywhere else log out here"